spec/publisher/removing_content_with_gone_spec.rb

```diff
@@ -0,0 +1,52 @@
+feature "Removing content by returning gone from Publisher", publisher: true do
```

Comment:
Sorry nitpicky - but I'm not sure this title makes a huge amount of sense. It doesn't seem an easy thing to phrase about adding the gone but could be that you already caught it with `and_i_remove_the_published_artefact_with_no_redirect` What do you think to renaming file and scenario to removing content "without a redirect" so it's just the inverse of the "with redirect" behaviour?